content/blog/set-up-apache-skywalking-for-k8s-monitoring-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,374 @@
+---
+title: Set up Apache SkyWalking for K8s monitoring in HPE GreenLake for Private
+  Cloud Enterprise
+date: 2022-12-28T20:19:50.005Z
+author: Guoping JIA
+authorimage: /img/guoping.png
+disable: false
+---
+## Introduction
+
+[HPE GreenLake for Private Cloud Enterprise](https://www.hpe.com/us/en/greenlake/private-cloud-enterprise.html) delivers a modern private cloud to support your app workloads running in any combination across your edges, colocations, and data centers. It contains one HPE service, called [HPE GreenLake for Containers](https://www.hpe.com/us/en/greenlake/containers.html), which provides an enterprise-grade container management service using open source Kubernetes. It allows customers to create a Kubernetes cluster, view details about existing clusters, and launch the service console. 
```

Comment:
Based on the guidelines [here](https://rndwiki-pro.its.hpecorp.net/display/HCSS/GreenLake+product+naming+conventions) the naming convention for the containers cloud service within HPE GL4PCE is: HPE GreenLake for Private Cloud Enterprise: Containers

So, I think the second sentence should be:
it contains one HPE service, called [HPE GreenLake for Private Cloud Enterprise: Containers](https://www.hpe.com/us/en/greenlake/containers.html) that provides an enterprise-grade container....
The container service allows customers to create Kubernetes clusters, view details about existing clusters, and launch the service console.

Reply:
Good catch ! Changed it.